src/types.js

```diff
@@ -280,7 +280,8 @@ export type ServerInfo = {
 
 export type SettingsState = {
   maxPerPage: number,
-  singleServer: ?string
+  singleServer: ?string,
+  authMethods: string[]
```

Comment:
We could create an `AuthMethod` type which would be `"basicauth" | "fxa" | "ldap"`, and here have `AuthMethod[]`